<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="textedit_Test">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>221</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>64</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_printTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_clearTextEdit">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Text Edit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setFont">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setFontItalic">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Italic</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_setFontColor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Font Color Red</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fontSizeUp">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_fontSizeDown">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Font Size Down</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
